gradle-guide-error-prone/src/test/java/com/palantir/gradle/guide/errorprone/taskexecution/ProjectLayoutFixesTest.java

```diff
@@ -160,13 +171,19 @@ abstract class CustomTaskAction implements Action<Task> {
             public void execute(Task task) {
                 // BUG: Diagnostic contains: Instead of `getProject().getProjectDir()`, do `getProjectLayout().getProjectDirectory().getAsFile()`
                 task.getProject().getProjectDir();
-                bad_helper(task);
+                badHelper(task);
             }
 
-            public void bad_helper(Task task) {
+            public void badHelper(Task task) {
                 System.out.println("I am a happy squirrel");
                 // BUG: Diagnostic contains: Instead of `getProject().getProjectDir()`, do `getProjectLayout().getProjectDirectory().getAsFile()`
                 task.getProject().getProjectDir();
+                badHelper2(task);
+            }
+
+            public void badHelper2(Task task) {
+                // BUG: Diagnostic contains: Instead of `getProject().getLayout()`, do `getProjectLayout()`
+                task.getProject().getLayout();
```

Comment:
curious why this can't be converted?

Reply:
talked offline, this is an `Action`, can't control instantiation, potentially could if it was already abstract, but it's rare enough that's just not worth trying to fix